App/Client.xcodeproj/project.pbxproj

```diff
@@ -2872,6 +2873,14 @@
 				minimumVersion = 5.10.2;
 			};
 		};
+		44EAE655287440D900DECF18 /* XCRemoteSwiftPackageReference "GuardianConnect" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/GuardianFirewall/GuardianConnect";
+			requirement = {
+				kind = exactVersion;
+				version = 1.6.2;
+			};
+		};
```

Comment:
Is it necessary to add `GuardianConnect` to the project's SPM dependencies list?  I don't see any usages of the public GuardianConnect APIs in `AppDelegate`

Reply:
It did not compile without it, will remove and double check

Reply:
works without it, thanks for catching